AsyncDisplayKit.xcodeproj/xcshareddata/xcschemes/AsyncDisplayKit.xcscheme

```diff
@@ -75,6 +75,13 @@
             ReferencedContainer = "container:AsyncDisplayKit.xcodeproj">
          </BuildableReference>
       </MacroExpansion>
+      <EnvironmentVariables>
+         <EnvironmentVariable
+            key = "FB_REFERENCE_IMAGE_DIR"
+            value = "$(SOURCE_ROOT)/$(PROJECT_NAME)Tests/ReferenceImages"
+            isEnabled = "YES">
+         </EnvironmentVariable>
+      </EnvironmentVariables>
       <AdditionalOptions>
```

Comment:
Somehow my XCode doesn't recognize `FB_REFERENCE_IMAGE_DIR` preprocessor macro. I'm switching to declare it in our scheme. It's the recommended way and a bit easier to enable/disable.